samples/client/petstore/typescript-jquery/default/api/PetApi.ts

```diff
@@ -49,19 +49,19 @@ export class PetApi {
     /**
      * 
      * @summary Add a new pet to the store
-     * @param body Pet object that needs to be added to the store
+     * @param pet Pet object that needs to be added to the store
      */
-    public addPet(body: models.Pet, extraJQueryAjaxSettings?: JQueryAjaxSettings): JQuery.Promise<
+    public addPet(pet: models.Pet, extraJQueryAjaxSettings?: JQueryAjaxSettings): JQuery.Promise<
```

Comment:
@janjongboom please run the https://github.com/OpenAPITools/openapi-generator/blob/master/bin/typescript-jquery-petstore-all.sh script, not the openapi3 version.

Reply:
Pushed.